frontend/src/components/Following/FollowingButton.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../styles/prelude.scss";
+
+
+.following-post
```

Comment:
A way to apply a common style to multiple elements and avoid rewriting code is to line them up as such:
```
.following-post, .following-user {
 color: rgb(25, 136, 240);
 outline: none;
 border: none;
 border-radius: 2px;
 text-transform: uppercase;
 cursor: pointer;
}
```
Such that your remaining code looks like this:
```
.following-post {
 background-color: transparent;
 padding: 5px 7px;
}

.following-user {
 background-color: rbg(40, 40, 40);
 padding: 5px 20px;
}
```